### PR TITLE
Update Unity scripts to use separate dlls instead of nupkg for UWP

### PR DESCRIPTION
**Pull request by AnastasiaKubova**
Things to consider before you submit the PR:

* [ ] ~Has `CHANGELOG.md` been updated?~
* [ ] ~Are tests passing locally?~
* [x] Are the files formatted correctly?
* [ ] ~Did you add unit tests if this modifies the Windows code?~
* [x] Did you test your change with either the sample apps that are included in the repository or with a blank app that uses your change?

## Description

Download right dlls in cake script.

## Related PRs or issues

[AB#77422](https://msmobilecenter.visualstudio.com/454a20a9-dfe6-4c1f-8ae8-417f76adb472/_workitems/edit/77422)

### CODE REVIEW

File: nuget-tools.cake [hunk 1]
```diff
@@ -11,7 +11,6 @@ using NuGet.Protocol.Core.Types;
 using NuGet.Versioning;
 
 const string PrivateNugetFeedUrl = "https://msmobilecenter.pkgs.visualstudio.com/_packaging/";
```
Comment (MatkovIvan) [inline nuget-tools.cake]: Can it be removed?

File: nuget-tools.cake [hunk 2]
```diff
@@ -11,7 +11,6 @@ using NuGet.Protocol.Core.Types;
...
-const string UwpIL2CPPJsonUrl = SdkStorageUrl + "Newtonsoft.Json.dll";
```
Comment (MatkovIvan) [inline nuget-tools.cake]: It's part of another PR, please exclude unrelated changes

File: dll-tools.cake [hunk 3]
```diff
@@ -0,0 +1,45 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```
Comment (MatkovIvan) [inline dll-tools.cake]: No need to create separate file for one function. Please move it to main script

File: build.cake [hunk 4]
```diff
@@ -197,6 +198,46 @@ class UnityPackage
     }
 }
 
+async Task ProcessDownloadUwpPackages() {
```
Comment (MatkovIvan) [inline build.cake]: Format

File: build.cake [hunk 5]
```diff
@@ -197,6 +198,47 @@ class UnityPackage
     }
 }
 
+async Task ProcessDownloadUwpPackages() 
+{
+
```
Comment (MatkovIvan) [inline build.cake]:
```suggestion
```

**Review by russelarms — APPROVED** [review]

File: nuget-tools.cake [hunk 6]
```diff
@@ -79,38 +78,6 @@ async Task ProcessDependency(NuGetDependency dependency, string destination)
...
-    var tempContentPath = ExternalsFolder + moniker + "/";
-    DeleteDirectoryIfExists(tempContentPath);
-    Unzip(nupkgPath, tempContentPath);
-    ProcessAppCenterAssemblies(tempContentPath, destination);
-    DeleteFiles(nupkgPath);
-}
-
-string GetPrivateNuGetPackage(string packageId, string packageVersion)
-{
-    var nugetUser = EnvironmentVariable("NUGET_USER");
```
Comment (MatkovIvan) [inline nuget-tools.cake:94]: Please make sure that CI is ready for this change
